### DISCUSSION TITLE
why did the banner change?

### DISCUSSION THREAD

**Discussion by whyakari** [RuriOSS/daijin#34] [General]
### current
> 
![Screenshot_20230510-231957](https://github.com/Moe-hacker/termux-container/assets/58480908/a5be7ab1-e15d-4875-ad5e-f3b763312c73)

----

### banner I wanted back:
> ![Screenshot_20230510-212146](https://github.com/Moe-hacker/termux-container/assets/58480908/9c1fb709-ebaa-4b4c-ae6c-88bb92d1dc30)

**Comment by Moe-hacker** [comment]
Yeah, it's back now.
